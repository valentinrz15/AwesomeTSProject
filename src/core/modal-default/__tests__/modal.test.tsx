import React from 'react';
import renderer from 'react-test-renderer';
import { ModalDefault } from '../index';

describe('modal default test', () => {
  it('should render modal default snapshot', () => {
    const render = renderer.create(<ModalDefault />).toJSON();

    expect(render).toMatchSnapshot();
  });
});
