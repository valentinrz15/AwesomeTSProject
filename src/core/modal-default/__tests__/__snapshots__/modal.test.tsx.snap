// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modal default test should render modal default snapshot 1`] = `
<Modal
  hardwareAccelerated={false}
  style={
    Object {
      "flex": 1,
    }
  }
  visible={true}
>
  <Text>
    Modal
  </Text>
</Modal>
`;
